package.json

```diff
@@ -50,5 +50,8 @@
   "mocha": {
     "timeout": 11000,
     "reporter": "spec"
+  },
+  "dependencies": {
+    "win-node-env": "^0.6.1"
```

Comment:
@tranjennie Shouldn't this just be installed globally to your local node env not to the project?

Reply:
Will remove this